client/src/components/App.jsx

```diff
@@ -76,4 +76,4 @@ class App extends React.Component {
   }
 }
 
-export default App;
\ No newline at end of file
+export default Recommendation;
```

Comment:
Changing the class name from App to Recommendation is okay but not necessary, specifying which id you want to render it to is the most important part :D